api/tests/unit/application/security-pre-handlers_test.js

```diff
@@ -579,4 +580,62 @@ describe('Unit | Application | SecurityPreHandlers', function () {
       });
     });
   });
+
+  describe('#checkUserIsAdminInOrganizationOrOwnerOfTheCampaign', function () {
+    context('Successful case', function () {
+      it('should authorize access to resource when the user is authenticated and is admin in organization or owner of the campaign', async function () {
```

Comment:
Il y a plus d'un cas de succès
- l'utilisateur est admin et est propriétaire de la campagne
- l'utilisateur est admin mais n'est pas propriétaire
- l'utilisateur est membre et est propriétaire

On laisse la responsabilité au test unitaire du usecase et on dit qu'ici on ne fait qu'un seul cas passant ?

Si oui, je suggère de changer le titre du test par `should authorize access to resource when the user is authenticated and is admin in organization and is owner of the campaign` car dans le given l'userId et le owerId sont les mêmes

Reply:
On garde effectivement un cas passant donc j'ai corrigé le nom du test